locales/id/translation.json

```diff
@@ -151,6 +151,7 @@
   "However,\n\t\t\t\tthere are some overarching goals, both medium-term and long-term, that\n\t\t\t\thave been agreed upon by core developers:": "Namun,\n\t\t\t\tterdapat beberapa tujuan besar, baik jangka pendek maupun panjang, yang\n\t\t\t\ttelah disepakati oleh para pengembang inti:",
   "Roadmap": "Haluan",
   "Reporting Issues": "Melaporkan Masalah",
+  "Where?": "Di mana?"
```

Comment:
Missing comma. (Edit: fixed in the meantime)

---

We need a CI pipeline to check this.